db/migrate/20200311133454_add_author_to_cards.rb

```diff
@@ -0,0 +1,5 @@
+class AddAuthorToCards < ActiveRecord::Migration[5.2]
```

Comment:
@MattBudz as a general rule we don't slip changes to the DB as part of individual tasks w/o previous discussion.

Can you remove this (and the db/schema.rb) changes from here?

Maybe we need to open a separate convo for what to do with the Card's author.